icomoon/demo-files/demo.js

```diff
@@ -0,0 +1,30 @@
+if (!('boxShadow' in document.body.style)) {
+	document.body.setAttribute('class', 'noBoxShadow');
+}
+
+document.body.addEventListener("click", function(e) {
+	var target = e.target;
+	if (target.tagName === "INPUT" &&
+		target.getAttribute('class').indexOf('liga') === -1) {
+		target.select();
+	}
+});
+
+(function() {
+	var fontSize = document.getElementById('fontSize'),
```

Comment:
I'd prefer not to touch any of the files under the icomoon folder, they are generated here: https://icomoon.io/